src/module/system/settings/index.ts

```diff
@@ -88,13 +88,19 @@ export function registerSettings(): void {
         },
     });
 
-    game.settings.register("pf2e", "enabledRulesUI", {
-        name: "PF2E.SETTINGS.EnabledRulesUI.Name",
-        hint: "PF2E.SETTINGS.EnabledRulesUI.Hint",
+    game.settings.register("pf2e", "minimumRulesUI", {
+        name: "PF2E.SETTINGS.MinimumRulesUI.Name",
+        hint: "PF2E.SETTINGS.MinimumRulesUI.Hint",
         scope: "world",
         config: true,
-        default: false,
-        type: Boolean,
+        default: 4,
```

Comment:
Wouldn't it make more sense to default to `3` here considering that you migrate "no players access" to "Assistant DM access"? This would also go in line with the Foundry defaults of Assistant DMs having the same level of permissions than full DMs?

Reply:
GM*, but it would.